neutron/neutron.go

```diff
@@ -51,12 +58,45 @@ func New(client client.Client) *Client {
 	return &Client{client}
 }
 
-// ListNetworksV2 gives details on available networks
-func (c *Client) ListNetworksV2() ([]NetworkV2, error) {
+// ----------------------------------------------------------------------------
+// TODO(hml): copied from the nova package.  However it should really be pulled out
+// and shared between goose pkgs, however we don't want to break compatibility at this time.
+//
+// Filtering helper.
+//
+// Filter builds filtering parameters to be used in an OpenStack query which supports
```

Comment:
please move the "Filter builds ..." to the top, so the docs lead with that, per standard go docs practices